api/templates/additional-comments.xml

```diff
@@ -0,0 +1,5 @@
+<AdditionalComments Version="1" Type="Pooled">
```

Comment:
I'd probably just put the whole thing in a conditional block that checks if Yes and comments is not empty string. It's a minor detail, but:
* that avoids unnecessary churn on test case files (so fewer things to have to test in eQIP, fewer known good files will have been modified)
* Skips empty element cruft
* Not sure if you've tested on e-QIP or not, but occasionally it complains about useless elements (but sometimes it ignores them too).